.ci/pipelines/build.Jenkinsfile

```diff
@@ -34,21 +34,16 @@ pipeline {
                         sh 'make -C .ci TARGET=ci ci'
                     }
                 }
-                stage('build') {
-                    failFast true
-                    parallel {
-                        stage("build and push operator image") {
-                            steps {
-                                sh '.ci/setenvconfig build'
-                                sh 'make -C .ci license.key TARGET=build-operator-multiarch-image ci'
-                            }
-                        }
-                        stage("build and push operator image in FIPS mode") {
-                            steps {
-                                sh '.ci/setenvconfig build'
-                                sh 'make -C .ci license.key TARGET=build-operator-multiarch-image ci ENABLE_FIPS=true'
-                            }
-                        }
+                stage("build and push operator image") {
+                    steps {
+                        sh '.ci/setenvconfig build'
+                        sh 'make -C .ci license.key TARGET=build-operator-multiarch-image ci'
+                    }
+                }
+                stage("build and push operator image in FIPS mode") {
+                    steps {
+                        sh '.ci/setenvconfig build'
+                        sh 'make -C .ci license.key TARGET=build-operator-multiarch-image ci ENABLE_FIPS=true'
```

Comment:
By default, parallel stages run on the same Jenkins worker. By declaring an agent for each step, the two stages will be run on different workers. 

```yaml
parallel {
    stage("build and push operator image") {
        agent {
            label 'linux'
        }
        steps {
            sh '.ci/setenvconfig build'
            sh 'make -C .ci license.key TARGET=build-operator-multiarch-image ci'
        }
    }
    stage("build and push operator image in FIPS mode") {
        agent {
            label 'linux'
        }    
        steps {
            sh '.ci/setenvconfig build'
            sh 'make -C .ci license.key TARGET=build-operator-multiarch-image ci ENABLE_FIPS=true'
        }
    }
}
```

Reply:
Thanks @thbkrkr , I wasn't aware of this.  I've updated to run on different agents instead to see if that resolves the issues.